lib/emoji.js

```diff
@@ -1,5 +1,6 @@
 /*jslint node: true*/
 require('string.prototype.codepointat');
+var toArray = require('lodash/toArray');
```

Comment:
@shivkanthb I see, in that case, it might be best to just use https://www.npmjs.com/package/lodash.toarray (so that we don't have to have all of lodash as dependency)